vassal-app/src/main/resources/VASSAL/i18n/Editor.properties

```diff
@@ -654,6 +654,32 @@ Editor.Stacking.color_nonexpand=Color of pieces when not expanded
 Editor.Stacking.component_type=Stacking options
 
 
+#Global Key Command
+Editor.GlobalKeyCommand.global_key_command=Global Key Command:  
```

Comment:
No More colons or white space at the end of prompt strings. We will sort that out in the configurers.